test/registry/SimpleAddressRegistry.t.sol

```diff
@@ -109,21 +109,4 @@ contract SimpleAddressRegistryTest is Test {
         assertEq(registry.get("FoundationUpgradeSafe"), 0xDEe57160aAfCF04c34C887B5962D0a69676d3C8B, "20");
         assertEq(registry.get("SecurityCouncil"), 0xf64bc17485f0B4Ea5F06A96514182FC4cB561977, "30");
     }
-
-    function test_hardcodedAddresses_opMainnet() public {
```

Comment:
Removing this test as I don't really see how it makes sense. I think addresses.toml shouldn't contain chain specific addresses e.g. nested safes. This PR opts to add nested addresses in the config.toml file of each test.

Reply:
Makes sense